File: cassandra/jruby/read/cassandra_read.rb
#! /usr/bin/jruby
# -*- coding: utf-8 -*-
#
#	cassandra_read.rb
#
#					Sep/06/2013
#
# ---------------------------------------------------------------------
include	Java
import	java.util.HashMap

import	java.sql.Connection
import	java.sql.DriverManager


load '/var/www/data_base/common/ruby_common/text_manipulate.rb'
load '/var/www/data_base/common/jruby_common/cassandra_manipulate.rb'
# ---------------------------------------------------------------------
puts "*** 開始 ***"

str_connect = "jdbc:cassandra://localhost:9160/city"
conn = DriverManager.getConnection (str_connect)

dict_aa = cassandra_to_dict_proc(conn)

conn.close()

dict_display_proc(dict_aa)

puts "*** 終了 ***"

# ---------------------------------------------------------------------

File: cassandra/rhino/read/cassandra_read.js
#! /usr/bin/java org.mozilla.javascript.tools.shell.Main
// ------------------------------------------------------------------
//
//	cassandra_read.js
//
//						Sep/05/2013
//
// ------------------------------------------------------------------
importPackage(java.lang);

importPackage(java.sql);

load ("/var/www/data_base/common/rhino_common/cassandra_manipulate.js");
load ("/var/www/data_base/common/rhino_common/text_manipulate.js");
// ---------------------------------------------------------------------- 
print ("*** 開始 ***");

var str_connect = "jdbc:cassandra://localhost:9160/city";
var conn = DriverManager.getConnection (str_connect)

var dict_aa = cassandra_to_dict_proc (conn)

dict_display_proc (dict_aa);

print ("*** 終了 ***");
// ---------------------------------------------------------------------- 
